cloud_functions/api_router/main.py

```
@@ -448,0 +1,448 @@
+            level="ERROR",
+            traceback=traceback.format_exc(),
+        )
+        return https_fn.Response(
+            status=500,
+            response=str(e),
+            headers={"Access-Control-Allow-Origin": "*"},
+        )
+
+
+@functions_framework.http
+def api_router(request):
```

Ruff (0.8.2)

`api_router` is too complex (12 > 10)

(C901)